packages/openneuro-app/src/scripts/search/filters-block-container.tsx

```diff
@@ -25,12 +25,26 @@ const FiltersBlockContainer: FC<FiltersBlockContainerProps> = ({
   const { searchParams, setSearchParams } = useContext(SearchParamsCtx)
   const selectedParams = getSelectParams(searchParams)
 
-  const noFilters = !useCheckIfParamsAreSelected(["modality_selected"])
+  const noFilters = !useCheckIfParamsAreSelected([
+    "modality_selected",
+    "brain_initiative",
+  ])
 
   const navigate = useNavigate()
   const { path } = useParams()
   const globalSearchPath = "/search"
 
+  const pattern = "/search/modality/*"
+
+  const convertWildcardToRegex = (pattern) => {
+    const escapedPattern = pattern.replace(/[|\\{}()[\]^$+?.]/g, "\\$&")
+    const regexPattern = "^" + escapedPattern.replace(/\*/g, ".*") + "$"
+    return new RegExp(regexPattern)
+  }
+
+  const regex = convertWildcardToRegex(pattern)
+  const isModalityPath = regex.test(location.pathname)
```

Comment:
I think useMatch() would be simpler, it will identify modality paths without the regular expressions and in an injection safe way.

Reply:
that is helpful

Reply:
updated